source/OutfitterPanel.cpp

```diff
@@ -124,8 +124,11 @@ int OutfitterPanel::DrawPlayerShipInfo(const Point &point)
 {
 	shipInfo.Update(*playerShip, player, collapsed.count("description"));
 	shipInfo.DrawAttributes(point);
+	const int attributesHeight = shipInfo.AttributesHeight();
+	const Point outfPoint(point.X(), point.Y() + attributesHeight);
+	shipInfo.DrawOutfits(outfPoint);
```

Comment:
```suggestion
	shipInfo.DrawOutfits({point.X(), point.Y() + attributesHeight});
```
not sure if I did this right, not doing this often, alternative:
```suggestion
	shipInfo.DrawOutfits(Point(point.X(), point.Y() + attributesHeight));
```

reason: no need to create that variable